From 395ca635fc7134957a17087384b6280007867f4c Mon Sep 17 00:00:00 2001
From: Vidya Velumuri <vidya.velumuri@cavium.com>
Date: Fri, 8 Dec 2017 10:28:00 +0530
Subject: [PATCH 262/375] octeontx-dpi: Use of actual correct macro
 DPI_DMA_REQQ_INT

Replaced the use of DPI_REQQ_INT with DPI_DMA_REQQ_INT
at proper places.

Signed-off-by: Vidya Velumuri <vidya.velumuri@cavium.com>
---
 drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c b/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c
index 55f54a1c3ddf..383f2a90f4f6 100644
--- a/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c
+++ b/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c
@@ -418,14 +418,17 @@ static irqreturn_t dpi_pf_intr_handler (int irq, void *dpi_irq)
 			i, reg_val);
 		dpi_reg_write(dpi, DPI_DMA_CCX_INT(i), 0x1ULL);
 	} else if (i < DPI_DMA_INT_REG) {
-		reg_val = dpi_reg_read(dpi, DPI_REQQX_INT(i - DPI_REQQ_INT));
-		dev_err(&dpi->pdev->dev, "DPI_REQQ_INT raised for q:%d: 0x%016llx\n",
+		reg_val = dpi_reg_read(
+			dpi, DPI_REQQX_INT(i - DPI_DMA_REQQ_INT));
+		dev_err(&dpi->pdev->dev,
+			"DPI_REQQ_INT raised for q:%d: 0x%016llx\n",
 			(i - 0x40), reg_val);
 
-		dpi_reg_write(dpi, DPI_REQQX_INT(i - DPI_REQQ_INT), reg_val);
+		dpi_reg_write(
+			dpi, DPI_REQQX_INT(i - DPI_DMA_REQQ_INT), reg_val);
 
 		if (reg_val & (0x71ULL))
-			dpi_queue_reset(dpi, (i - DPI_REQQ_INT));
+			dpi_queue_reset(dpi, (i - DPI_DMA_REQQ_INT));
 	} else if (i == DPI_DMA_INT_REG) {
 		reg_val = dpi_reg_read(dpi, DPI_INT_REG);
 		dev_err(&dpi->pdev->dev, "DPI_INT_REG raised: 0x%016llx\n",
@@ -839,4 +842,3 @@ static void __exit dpi_cleanup_module(void)
 
 module_init(dpi_init_module);
 module_exit(dpi_cleanup_module);
-
-- 
2.14.1

